README

```diff
@@ -968,18 +968,21 @@ INSTALLATION OPTIONS
 --enable-dlopen
   Build all of Open MPI's components as standalone Dynamic Shared
```

Comment:
This line has never actually been true.  --enable-dlopen has always just allowed for component loading at runtime.  It was never the cause of the default behavior.  It's definitely not true now.